newIDE/app/src/MainFrame/index.js

```diff
@@ -522,13 +521,13 @@ const MainFrame = (props: Props) => {
   React.useEffect(
     () => {
       if (initialDialog === 'subscription') {
-        openSubscriptionDialog(true);
+        openSubscriptionDialog({ reason: 'Landing dialog at opening' });
```

Comment:
maybe "initial dialog"

Reply:
It's a bit generic. What is bothering you with this formulation?

Reply:
I didn't understand it at first reading, I had to look at where it's used in the code to understand.
But I guess once you know it it's fine